README.md

```diff
@@ -131,19 +131,19 @@ There are no ground truths of 4K anime.
 
 [Story Time](FAQ_Detail.md)
 
-Comparing PSNR/SSIM on 480p->720p upscales does not prove and is not a good indicator of 1080p->2160p upscaling quality. (Eg. poor performance of waifu2x on 1080p anime) 480p anime images have a lot of high frequency information (lines might be thinner than 1 pixel), while 1080p anime images have a lot of redundant information. 1080p->2160p upscaling on anime is thus objectively easier than 480p->720p.
+Comparing PSNR/SSIM on 480p->720p upscales does not prove and is not a good indicator of 1080p->2160p upscaling quality. (Eg. the poor performance of waifu2x on 1080p anime) 480p anime images have a lot of high-frequency information (lines might be thinner than 1 pixel), while 1080p anime images have a lot of redundant information. 1080p->2160p upscaling on anime is thus objectively easier than 480p->720p.
 
 ### I think the results are worse than \<x>
 
 Surely some people like sharper edges, some like softer ones. Do try it yourself on a few anime before reaching a definite conclusion. People *tend* to prefer sharper edges. Also, seeing the comparisons on a 1080p screen is not representative of the final results on a 4K screen, the pixel density and sharpness of the final image is simply not comparable.
 
-### Note for those who think this is not a 'upscaling' algorithm.
+### Note for those who think this is not an 'upscaling' algorithm.
 
 [Explanation](Upscaling.md)
 
 TL;DR
 
-Sharpening, De-Blurring and Super-Resolution are equivalent.  
-Anime4K can de-blur, and is equivalent to a SR algorithm.  
+Sharpening, De-Blurring, and Super-Resolution are equivalent.  
+Anime4K can de-blur, and is equivalent to an SR algorithm.  
```

Comment:
Line 147: Would be favourable to have "a Super-Resolution algorithm" rather than causing potential confusion